splunk_connector.py

```diff
@@ -90,22 +91,31 @@ def _get_error_message_from_exception(self, e):
         error_code = None
         error_msg = consts.SPLUNK_ERR_MSG_UNAVAILABLE
 
+        self._dump_error_log(traceback.format_stack(e))
```

Comment:
does this work? I think the [traceback](https://docs.python.org/3/library/traceback.html#traceback.format_stack) functions assume they're called within an exception handler so you don't need to pass it any parameters

Reply:
Yes, this also works fine. The reason we need to pass the exception object is because we are using the _dump_error_log() function. However, I changed this line to print a simple traceback using the error_print() function.